tests/visualization_tests/test_pareto_front.py

```diff
@@ -13,12 +15,33 @@
 from optuna.trial import TrialState
 from optuna.visualization import plot_pareto_front
 from optuna.visualization._pareto_front import _make_hovertext
+from optuna.visualization._plotly_imports import go
+
+
+def _check_data(figure: "go.Figure", axis: str, expected: Sequence[int]) -> None:
+    """Compare `figure` against `expected`.
+
+    Concatenate `data` in `figure` in reverse order, pick the desired `axis`, and compare with
+    the `expected` result.
+
+    Args:
+        figure (:class:`go.Figure`): A figure.
+        axis (str): The axis to be checked.
+        expected (Sequence[int]): The expected result.
```

Comment:
Types are inferred from the type hints in the signature, so you can skip them here, although this matter less given this is a private function.
```suggestion
        figure: A figure.
        axis: The axis to be checked.
        expected: The expected result.
```